vnext/Microsoft.ReactNative.Cxx/JSValueReader-WinRT.h

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <CppWinRTIncludes.h>
+#include <UI.Xaml.Markup.h>
+#include <chrono>
+
+#include "JSValue.h"
+#include "JSValueTreeReader.h"
+#include "StructInfo.h"
+
+#include "winrt/Microsoft.ReactNative.h"
+
+#include <string>
+
+namespace winrt::Microsoft::ReactNative {
+inline BYTE GetAFromArgb(DWORD v) {
```

Comment:
>BYTE [](start = 7, length = 4)

Please use standard C++ types instead of Windows-specific types. E.g. uint8_t or byte instead of BYTE and uint32_t instead of DWORD. While we can see the old-style Windows macros inside of API docs, the C++/WinRT is specifically all about using the standard C++ types. I.e. base.h and all generated C++/WInRT code uses only C++ standard types.
For example, from base.h:
constexpr std::array<uint8_t, 8> size_to_bytes(size_t size) noexcept